core/frontend/src/extension/ExtensionHost.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Extensions
+ */
+
+import { IModelApp } from "../IModelApp";
+import type { ToolAdmin } from "../tools/ToolAdmin";
+import type { AccuSnap } from "../AccuSnap";
+import type { NotificationManager } from "../NotificationManager";
+import type { ViewManager } from "../ViewManager";
+import type { ElementLocateManager } from "../ElementLocateManager";
+import type { Localization } from "@itwin/core-common";
+import type { RenderSystem } from "../render/RenderSystem";
+
+/**
+ * Subset of IModelApp exposed to Extensions
+ * @beta
+ */
+export class ExtensionHost {
+  protected constructor() { }
+
+  public static get toolAdmin(): ToolAdmin { return IModelApp.toolAdmin; }
+  public static get notifications(): NotificationManager { return IModelApp.notifications; }
+  public static get viewManager(): ViewManager { return IModelApp.viewManager; }
+  public static get locateManager(): ElementLocateManager { return IModelApp.locateManager; } // internal ?
+  public static get accuSnap(): AccuSnap { return IModelApp.accuSnap; }
+  public static get localization(): Localization { return IModelApp.localization; } // re think this, should be contribution point
```

Comment:
do we want to open this can of worms? Is it not easier (and fair) for consumers to provide already localized strings (if they care to)

Reply:
I'm debating on this one... The localization interface exposes a lot more than I'd like since it does have implications and side-effects.

Maybe we expose a limited API for `getLocalizedString`? That way an extension today could get them?  We'd need to quickly come up with a contribution point for adding localized strings though.

Reply:
removed for now

Reply:
@kckst8 are you going to pick up removing this from the initial API?

Reply:
Already removed it from the host API and it doesn't appear to be exported for the extension runtime. Am I missing somewhere?

Reply:
Nope, you're good. GitHub just hadn't updated for me...